app.js

```diff
@@ -1,7 +0,0 @@
-const logger = require('@pubsweet/logger')
```

Comment:
what's the thinking behind deleting this? we need it to start the app.

Reply:
Do we? .... where is it called from? Everywhere I look the app is started via `pubsweet server`.

Reply:
https://gitlab.coko.foundation/yld/xpub-deployment-config-postgres/blob/master/templates/xpub-elife-deployment.yaml#L75

Reply:
I think this also is another good reason we need an explanation / documentation on how the deployment process works :)

Reply:
`pubsweet server` starts the process with forever, this is intended for production environments where we do not know how the app will be started.

Reply:
... can you expand on "where we do not know how the app will be started" please